src/components/ProgressIndicator/ProgressIndicator.tsx

```diff
@@ -107,8 +111,14 @@ export const ProgressIndicator: ChakraComponent<
                 </ChakraCircularProgressLabel>
               )}
             </ChakraCircularProgress>
-            {showLabel && size !== "small" && (
-              <Label id={`${id}-label`} htmlFor={id}>
+            {showLabel && (size !== "small" || labelPlacement === "right") && (
```

Comment:
Can you add a few more examples in the storybook doc section? The current example shows size="small" and labelPlacement="right", but it'd be good to see others such as the regular size and labelPlacement="right".